### PR TITLE
Migrate Project#users page to bootstrap

### PR DESCRIPTION
**Pull request by eduardoj**
project_users

### CODE REVIEW

File: src/api/app/mixins/webui/manage_relationships.rb [hunk 1]
```diff
@@ -14,11 +14,7 @@ def save_person
     rescue NotFoundError,
            Relationship::SaveError => e
       flash[:error] = e.to_s
-      if params[:webui2]
-        redirect_to users_path
-      else
-        redirect_to add_path(:add_person)
-      end
+      redirect_to add_path(:add_person)
```
Comment (dmarcoux) [inline src/api/app/mixins/webui/manage_relationships.rb]: It's like this now because package and project both have a Bootstrap version, right?

Comment (eduardoj) [reply src/api/app/mixins/webui/manage_relationships.rb]: > It's like this now because package and project both have a Bootstrap version, right?

Yes.

**Comment by eduardoj** [comment]
[Users](https://trello.com/c/78yGAxL5/967-users)

**Comment by eduardoj** [comment]
@dmarcoux:

> The breadcrumbs are missing.

Right. Added. Please re-review.

**Pull request merged**